tribler.sh

```diff
@@ -27,7 +27,8 @@ if [ "$UNAME" = "Linux" ]; then
     }
 
     python2.7 $TRIBLER_SCRIPT
-
+elif [[ "$UNAME" = CYGWIN_NT* ]]; then
+    python $TRIBLER_SCRIPT
```

Comment:
However, that double square bracket thingy does not work with all versions of bash, I will rewrite it to use `grep` which is more universal.